.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -46,6 +46,23 @@ None
 If tests aren't applicable just write N/A.
 -->
 
+#### What user story does this cover?
+
+<!-- Please add a reference to the user story that this feature would cover
+
+The existing user stories exist in docs/user-stores.md
+
+If the user story doesn't exist yet, please add it to the main document
+to help people understand what the feature is trying to solve and how it's
+meant to help the target personas.
+
+If this would help a target persona that's not currently listed, please add it
+to the document in docs/personas.md
+
+If this PR doesn't refer to a user story, adding N/A is acceptable.
+ -->
+
+
```

Comment:
Not sure if we need to add it to the PR template.

Reply:
I thought it would be a good idea since it might encourage people to revisit the user stories and thus help them re-think how their code is meant to help folks. Let's get more input on this. If more folks aren't keen on adding it to the PR template, we can remove it.

Reply:
That's an interesting point. Assuming that meaningful PRs would always be linked to an issue, this extra link to user stories may be redundant.

Thinking out loud, Issues generally defines the `what` and `why` of a change - the latter point now being linked to an user story. Now the PRs  defines the `how`, which should be linked to an issue giving us already the full chain. But happy to hear other views.

Reply:
The reasoning makes sense. And yeah, if the user story is already referenced in the GH issue, then definitely it's not needed here in the PR template. remove it.